<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginDialog</string>
  </property>
  <widget class="QTabWidget" name="tb_page">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>211</width>
     <height>231</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;登录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pg_login">
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QLabel" name="lb_tel_login">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password_login">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel_login">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>12345678912</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password_login">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>222</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear_login">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>150</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit_login">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_register">
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLabel" name="lb_tel_register">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>手机号</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password_register">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_nicname_register">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>昵称</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_tel_register">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password_register">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nicname_register">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_clear_register">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_commit_register">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_icon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/images/register.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
